File: level30/homework/hw6.py
# 6) შექმენით სტრინგებით სავსე სია, წაშალეთ ის სტრინგ მონაცემთა ტიპის ელემენტები რომლებიც არიან 5-ზე მეტი სიგრძეში ან დგანან კენტ ინდექსზე. გამოიყენეთ remove() ფუნქცია.

strings = ["zaza", "giorgi", "saba", "luka","levani","nika"]

i = 0
while i < len(strings):
    if len(strings[i]) > 5 or i % 2 != 0:
        strings.remove(strings[i])
    else:
        i += 1

print(strings)

File: hw6.js
// 6)მოცემულია:
// let students = [
//     ["Giorgi", 18],
//     ["Nika", 20],
//     ["Luka", 17],
//     ["Saba", 19]
// ];
// შექმენი ფუნქცია getStudent(students), რომელმაც უნდა:

// გამოიტანოს პირველი სტუდენტის სახელი;
// გამოიტანოს მეორე სტუდენტის ასაკი;
// შეცვალოს მესამე სტუდენტის ასაკი 18-ზე;
// დააბრუნოს განახლებული სია.

let students = [["Giorgi", 18],["Nika", 20],["Luka", 17],["Saba", 19]]

function getStudent(students) {
    console.log(students[0][0])
    console.log(students[1][1])
    students[2][1] = 18
    return students
}